lib/generators/maintenance_tasks/task_generator.rb

```diff
@@ -9,23 +9,29 @@ class TaskGenerator < Rails::Generators::NamedBase
     desc 'This generator creates a task file at app/tasks and a corresponding '\
       'test.'
 
-    class_option :csv, type: :boolean, default: false,
-      desc: 'Generate a CSV Task.'
+    TASK_TYPES = ['collection', 'csv'].freeze
```

Comment:
I don't think `.freeze` is necessary 😄

Reply:
I'm just used to freezing all mutable constants (so they're actually "constant"). Do you see a use case for modifying it?

Reply:
The argument I've heard from Rafael & others is that using `.freeze` is defensive and that we should assume developers are well-intended with the code they write instead of protecting against something that they likely won't do. The fact that it only warns instead of raising if you redefine the constant means that we're not really protected from mutation anyways. We haven't frozen any of our other constants in the gem for this reason 😄